### PR TITLE
Added ingress class name option to avoid using deprecated annotation

### PR DESCRIPTION
**Pull request by kflorianski-ibm**

### CODE REVIEW

File: bundle/manifests/ibm-licensing-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -26,8 +26,8 @@ metadata:
     capabilities: Seamless Upgrades
     categories: Monitoring
     certified: "false"
-    containerImage: icr.io/cpopen/ibm-licensing-operator:4.2.5
-    createdAt: "2024-05-10T11:03:47Z"
+    containerImage: icr.io/cpopen/ibm-licensing-operator:4.2.4
```
Comment (Pejdzor) [inline bundle/manifests/ibm-licensing-operator.clusterserviceversion.yaml:29]: why you bumped down image version?

**Review by Pejdzor — CHANGES_REQUESTED** [review]

File: bundle/manifests/ibm-licensing-operator.clusterserviceversion.yaml (hunk 1 above)
Comment (kflorianski-ibm) [reply bundle/manifests/ibm-licensing-operator.clusterserviceversion.yaml]: Fixed in dd10f5d8d1b4168ab92d889358c2c8529ce33482

**Review by Pejdzor — APPROVED** [review]
Look's good to me :)

**Review by ljeda — APPROVED** [review]

**Comment by ljeda** [comment]
does this item require any documentation changes?

**Comment by kflorianski-ibm** [comment]
@ljeda Yes, I have them ready. Will prepare once tests for this are finished

**Comment by michalpaluch** [comment]
approved

**Pull request merged**